public/app/plugins/panel/timeseries/migrations.test.ts

```diff
@@ -36,7 +54,16 @@ describe('Graph Migrations', () => {
       angular: stepedColordLine,
     };
     const panel = {} as PanelModel;
-    panel.options = graphPanelChangedHandler(panel, 'graph', old);
+    panel.options = graphPanelChangedHandler(panel, 'graph', old, prevFieldConfig);
+    expect(panel).toMatchSnapshot();
+  });
+
+  it('preserves colors in series overrides', () => {
```

Comment:
HyperNit™: "preserves colors in series overrides" suggests the series overrides will be still there(which is not true). Maybe "preserves colors from series overrides" ? 

Feel free to ignore ^^, I am not a native speaker:)

Reply:
yep i agree, that's more accurate